README.md

```diff
@@ -66,14 +69,15 @@ async function streamResolver(id, callback) {
 }
 ```
 
-### Dimension Callback
+### Dimension Function
 
 The calling function can also supply the processor with an optional Dimension
-callback that takes an ID and returns a `{width: w, height: h}` object. This
-allows for caching dimensions and avoiding an expensive image request.
+callback that takes information about the request [(`id` and `baseUrl`)](#id--baseurl) and returns a
+`{width: w, height: h}` object.
+This allows for caching dimensions and avoiding an expensive image request.
 
 ```javascript
-async function dimensionResolver(id, callback) {
```

Comment:
As far as I could tell, this function is not called with a `callback`, so I think this documentation wasn't correct previously.